cmd/root.go

```diff
@@ -92,9 +92,9 @@ Complete documentation is available at https://appsody.dev`,
 		//Run: no run action for the root command
 	}
 
-	rootCmd.PersistentFlags().StringVar(&rootConfig.CfgFile, "config", "", "config file (default is $HOME/.appsody/.appsody.yaml)")
-	rootCmd.PersistentFlags().BoolVarP(&rootConfig.Verbose, "verbose", "v", false, "Turns on debug output and logging to a file in $HOME/.appsody/logs")
-	rootCmd.PersistentFlags().BoolVar(&rootConfig.Dryrun, "dryrun", false, "Turns on dry run mode")
+	rootCmd.PersistentFlags().StringVar(&rootConfig.CfgFile, "config", "", "Location of Appsody config files to use. (default '$HOME/.appsody/.appsody.yaml')")
```

Comment:
Judging by the earlier conversation, it sounds like this is actually a path to a single file? Is it relative or absolute? Here's a possible alternative:
"The absolute path to the Appsody config file. Use this option when you want to specify your own, customized config file, instead of the default: '$HOME/.appsody/.appsody.yaml'."
(That's assuming there isn't an explanation nearby for when/why you would want to use this option :)